### DISCUSSION TITLE
Can't Verify My University Email (University of Bahrain)

### DISCUSSION THREAD

**Discussion by AbRahman-ra** [community/community#55284] [GitHub Education]
### I can't verify my university email 

As it shows, my university email address (@stu.uob.edu.bh) cannot be verified. GitHub doesn't send any verification instructions. Knowing that the email can receive emails outside of the domain normally.

Already checked spam and all of that. I have been more than 2 weeks trying to verify it, but I couldn't. What should I do? 

![Screenshot](https://github.com/community/community/assets/126220430/fecff033-434e-4aec-a949-42bed42f8405)

**Comment by Ntshangase** [comment]
Hi @AbRahman-ra 

It would be easier to help you if you show a screenshot of the error message( If-any ) that you get when you try verify your student email.

**Reply by AbRahman-ra** [reply]
Hello.

Sorry for unclarity, the issue simply is when I click on "Resend Verification," it's written on Github that the email is sent, but nothing is sent to my email. I did this many times and waited for many days, but still unsolved. I even tried to delete and add my email again but still didn't solve it.

I tried to send an email from my normal gmail and yahoo accounts and they were sent notmally. So I know my university email is not causing the problem.

**Reply by Ntshangase** [reply]
Please check this [this similar discussion ](https://github.com/orgs/community/discussions/52437) and try the recommended solution, Then tell me how that goes.

**Reply by AbRahman-ra** [reply]
This is not applicable, I still didn't submit any application. I need to add an academic email before going to the step mentioned in the solution, and that's the pain point here. I can't, why? Because it's not verified. Why it's not verified? Because I hadn't received any verification email yet.

![Problem](https://github.com/community/community/assets/126220430/14a34290-ee0b-482c-849c-0b0d1e1deddb)

**Reply by Ntshangase** [reply]
Ok, 
- Can you try logging out and in to GitHub again then try adding your email.

**Reply by AbRahman-ra** [reply]
I did now, but still nothing changed. I tried even removing and adding my university email again but still unsolved.
